tests/sentry/utils/test_snowflake.py

```diff
@@ -57,3 +63,27 @@ def test_out_of_region_sequences(self):
             generate_snowflake_id("test_redis_key")
 
         assert str(context.value) == "No available ID"
+
+    @freeze_time(CURRENT_TIME)
+    @mock.patch("sentry.utils.snowflake.get_region_mapping")
```

Comment:
How about using `with override_settings` as the test target?  Helps test deeper the load from settings mechanism as well.  Minor nit though.

Reply:
As I mentioned [here](https://github.com/getsentry/sentry/pull/39253#discussion_r980508451), `override_settings` isn't enough by itself, but I think the current implementation is as good.